articles/2024822_Daytona_Office_Hours_#9.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Daytona Office Hours #9 - Pre-Builds Demo & Latest Fixes!"
+description: "The recent updates and highlights shared during the Daytona Office Hours #9 and a special demo for Pre-Builds."
+date: 2024-8-22
+author: "Daytona"
+tags: ["Office Hours", "Prebuilds"]
+---
+Daytona-Office-Hours-%239
+# Daytona Office Hours #9 - Pre-Builds Demo & Latest Fixes!
+
+## Introduction
+
+In our recent office hours, we discussed the latest development news on Daytona. Even though we haven't had a new release yet, there are some interesting features and fixes we are working on, and looking forward to merging them.
+
+### TL;DR
+
+- **Pre-Builds demo**
+- **Recent fixes and future releases**
+- **Contributions and bounties**
+
+## Pre-Builds
+
+A couple of weeks back we introduced a major feature called [Project Configs](https://github.com/daytonaio/daytona/pull/789). They allow users to define project configurations in advance and use them when creating workspaces in the future. They contain the relevant repository and build information, environment variables which are then used by default whenever a project with the specified repository is created. They are part of a bigger effort to incorporate [pre-builds](https://github.com/daytonaio/daytona/pull/912) which will let users set up "ready-to-go" builds and spend less time waiting for their development environments to be ready. The documentation for the project config can be found [here](https://www.daytona.io/docs/reference/cli/#daytona-project-config).
+
+Prebuilds are a way to speed up development by setting a project configuration to listen to changes in the underlying repository and run a build whenever necessary to ensure that once the user wants to create a project and start working, the creation process is much quicker. The user can then decide on a commit interval after which a build should be triggered and any specific trigger files whose changes should immediately start the build process. Prebuilds is still under development, but it is almost ready to be merged. In the office hours video, we presented a demo of creating and managing pre-builds.
```

Comment:
too big paragraph make it more readable, remove:  In the office hours video, we presented a demo of creating and managing pre-builds.